tests/e2e/router.spec.ts

```diff
@@ -0,0 +1,162 @@
+import type { Page } from "@playwright/test";
+import { test, expect } from "@tests/support/fixtures.js";
+
+// Reloads the current page and verifies that the URL stays correct
+const reload = async (page: Page) => {
```

Comment:
Could we call this `expectUrlPersistsReload`?

Reply:
yeah sure 👍

Reply:
Fixed